_posts/2016-12-14-four-tips-sass-variables.markdown

```diff
@@ -0,0 +1,237 @@
+---
+layout: post
+title: "Four Tips for Using Sass Variables"
+date: 2016-12-14
+categories: front-end sass css
+author: Max Antonucci
+excerpt: Sass variables help keep your CSS efficient and flexible, but they're more complex than you may think. These are four basic tips to get the most out from your Sass variables.
+
+source: internal
+ext-url: none
+---
+
+While [Sass](http://sass-lang.com/) brings many great functions to CSS, one of the biggest is variables. Variables mean no more `ctrl-F`-ing our way through stylesheets when we change a color or margin value. Putting these frequent values in one place makes managing them that much easier.
```

Comment:
I think the markdown engine can handle `<kbd>CTRL</kbd>+<kbd>F</kbd>` syntax.

#### Example

<kbd>CTRL</kbd>+<kbd>F</kbd>